### ISSUE TITLE
RelatedDocuments

### ISSUE THREAD

**Issue by Henrich-BGRCI** [esdscom/sdscom-xml#177]
What are the differences between SpecialExtensions and RelatedDocuments?
Where is the datatype definition of "FileName" (used in RelatedDocuments and ExposureScenarioFile)?

**Comment by joern-pes** [comment]
Our interpretation of this is that RelatedDocuments are a collection of related documents like other pdf-files. SpecialExtensions are extensions of the XML structure which can the issuer define.

**Comment by dirk-qualisys** [comment]
I agree with the interpretation of Jörn; this is also what the annotations say. Any need for further action here?
